ui/app/AppLayouts/Communities/controls/TokenHolderListItem.qml

```diff
@@ -104,17 +172,17 @@ ItemDelegate {
             leftPadding: 0
             rightPadding: 0
             hoverEnabled: false
-            nickName: root.contactDetails.localNickname
-            userName: ProfileUtils.displayName("", root.contactDetails.ensName, root.contactDetails.displayName, root.contactDetails.alias)
-            pubKey: root.contactDetails.isEnsVerified ? "" : root.contactDetails.compressedPublicKey
-            isContact: root.contactDetails.isContact
-            isVerified: root.contactDetails.trustStatus === Constants.trustStatus.trusted
-            isUntrustworthy: root.contactDetails.trustStatus === Constants.trustStatus.untrustworthy
-            isAdmin: root.contactDetails.memberRole === Constants.memberRole.owner
-            status: root.contactDetails.onlineStatus
-            icon.name: root.contactDetails.thumbnailImage
-            icon.color: Utils.colorForPubkey(root.contactId)
-            ringSettings.ringSpecModel: Utils.getColorHashAsJson(root.contactId)
+            nickName: root.nickName
+            userName: root.userName
+            pubKey: root.isEnsVerified ? "" : root.compressedPubKey
+            isContact: root.isContact
+            isVerified: root.trustStatus === Constants.trustStatus.trusted
+            isUntrustworthy: root.trustStatus === Constants.trustStatus.untrustworthy
+            isAdmin: root.memberRole === Constants.memberRole.owner
```

Comment:
Are `admins` only the `owners`, or in this context yes?

Reply:
I just checked the code and indeed, it's about Owner. It's that condition that puts the crown 
```qml
            active: root.isAdmin
            sourceComponent: StatusIcon {
                anchors.verticalCenter: parent.verticalCenter
                icon: "crown"
                color: Theme.palette.directColor1
            }
```
I'll rename the property